src/phaser/scenes/board-scene.ts

```diff
@@ -1,20 +1,17 @@
-import AzNopolyPanel from "@/ui/panel";
-import GameBoard from "../board/board";
-import { BOARD_SIDE_LENGTH, HEIGHT, WIDTH } from "../main";
-import { FRAME_PADDING } from "../style";
-import { AzNopolyButton } from "../ui/button";
-import PlayerList from "../ui/player-list";
-import RandomSelectionWheel from "../ui/random-selection-wheel";
+import AzNopolyPanel from "@/phaser/components/ui/panel";
+import GameBoard from "../components/board";
+import { FRAME_PADDING } from "../../style";
+import { AzNopolyButton } from "../components/ui/button";
+import RandomSelectionWheel from "../components/ui/random-selection-wheel";
```

Comment:
```suggestion
import GameBoard from "@/components/board";
import { FRAME_PADDING } from "@/style";
import { AzNopolyButton } from "@/components/ui/button";
import RandomSelectionWheel from "@/components/ui/random-selection-wheel";
```

Reply:
Laziness + vscode seemingly chooses the import format randomly

Reply:
@ should be used